acp/dac/source_hub.go

```diff
@@ -346,7 +358,7 @@ func (a *SourceHubDocumentACP) AddActorRelationship(
 		return false, err
 	}
 
-	return cmdResult.GetResult().GetSetRelationshipResult().RecordExisted, nil
+	return cmdResult.GetResult().GetSetRelationshipResult().GetRecordExisted(), nil
```

Comment:
question: @iverc, wondering why this change was needed? and if this fixes something we also have a similar getter for the Delete Relationship, should it be done there as well? and also local acp has a similar getter too for both adding and deleting relationships.